lib/encodeImages.js

```diff
@@ -4,6 +4,12 @@ var Jimp = require('jimp');
 
 module.exports = encodeImages;
 
+/**
+ * Encode images to a glTF buffer
+ *
+ * @param {Object} gltf - An object holding a glTF hierarchy
+ * @param {Function} callback
```

Comment:
The believe callbacks are documented like this: http://usejsdoc.org/tags-callback.html. At the same time, we will eventually change the callbacks to promises anyways.